test/gtest/ucs/test_async.cc

```diff
@@ -416,12 +416,12 @@ UCS_TEST_P(test_async, max_events, "ASYNC_MAX_EVENTS=4") {
 
 UCS_TEST_P(test_async, many_timers) {
 
-    for (int count = 0; count < 1000; ++count) {
+    for (int count = 0; count < 4010; ++count) {
```

Comment:
magic numbers? :)

Reply:
to reach the limit of id's with the previous (buggy) implementation